src/MoBi.Core/Commands/RemoveTagConditionCommandBase.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using System;
```

Comment:
I would rename the file from REmoveTagConditionCommandBase to REmoveTagConditionCommand